<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageCliquableClass</class>
 <widget class="QMainWindow" name="ImageCliquableClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>717</width>
    <height>657</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImageCliquable</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QPushButton" name="Zone1">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>70</y>
         <width>81</width>
         <height>101</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0.3);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Zone2">
       <property name="geometry">
        <rect>
         <x>380</x>
         <y>80</y>
         <width>100</width>
         <height>100</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0.3);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="Image">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>601</width>
         <height>401</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap>chateau/IMG_2726.JPG</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QPushButton" name="Return">
       <property name="geometry">
        <rect>
         <x>530</x>
         <y>330</y>
         <width>75</width>
         <height>71</height>
        </rect>
       </property>
       <property name="text">
        <string>Return</string>
       </property>
      </widget>
      <widget class="QLafayetteMenu" name="widget_2" native="true">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>131</width>
         <height>601</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
      <widget class="QPushButton" name="Zone4">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>0</width>
         <height>0</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0.3);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Zone3">
       <property name="geometry">
        <rect>
         <x>110</x>
         <y>140</y>
         <width>41</width>
         <height>141</height>
        </rect>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgba(255, 255, 255, 0.3);</string>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QPushButton" name="Zone5">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>0</width>
         <height>0</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="Puzzle" name="widget_3" native="true">
       <property name="geometry">
        <rect>
         <x>109</x>
         <y>-1</y>
         <width>491</width>
         <height>401</height>
        </rect>
       </property>
      </widget>
      <zorder>Image</zorder>
      <zorder>Zone1</zorder>
      <zorder>Zone2</zorder>
      <zorder>Return</zorder>
      <zorder>widget_2</zorder>
      <zorder>Zone4</zorder>
      <zorder>Zone3</zorder>
      <zorder>Zone5</zorder>
      <zorder>widget_3</zorder>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QLafayetteMenu</class>
   <extends>QWidget</extends>
   <header>QLafayetteMenu.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>Puzzle</class>
   <extends>QWidget</extends>
   <header>Puzzle.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="ImageCliquable.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Zone2</sender>
   <signal>clicked(bool)</signal>
   <receiver>ImageCliquableClass</receiver>
   <slot>switchImage2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>389</x>
     <y>188</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Zone1</sender>
   <signal>clicked(bool)</signal>
   <receiver>ImageCliquableClass</receiver>
   <slot>switchImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>179</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Zone3</sender>
   <signal>clicked(bool)</signal>
   <receiver>ImageCliquableClass</receiver>
   <slot>switchImage3()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>159</x>
     <y>289</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Return</sender>
   <signal>clicked(bool)</signal>
   <receiver>ImageCliquableClass</receiver>
   <slot>previousImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>539</x>
     <y>374</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>198</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Zone4</sender>
   <signal>clicked(bool)</signal>
   <receiver>ImageCliquableClass</receiver>
   <slot>pickupKey()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>8</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>371</x>
     <y>311</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Zone5</sender>
   <signal>clicked(bool)</signal>
   <receiver>ImageCliquableClass</receiver>
   <slot>pickupSymbol()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>8</x>
     <y>8</y>
    </hint>
    <hint type="destinationlabel">
     <x>358</x>
     <y>259</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>switchImage()</slot>
  <slot>switchImage2()</slot>
  <slot>switchImage3()</slot>
  <slot>previousImage()</slot>
  <slot>pickupKey()</slot>
  <slot>pickupSymbol()</slot>
 </slots>
</ui>
